docs/en/api/QueryProductConfigs.md

```diff
@@ -121,19 +127,17 @@ A failed request can result in a response with one of these HTTP status values,
 
 _Note that server errors can occur that require exponential back-off on retry._
 
-***
+## <a name="queryIndividualProductConfigurations" class="api-ref-subtitle">Query Individual Product Profiles</a>
 
-## Query Individual Product Configurations
+A GET request to the **{orgId}/products/{productId}/configurations/{configId}** resource retrieves information for a defined product profile. The body of the response contains the configuration information in JSON format.
 
-A GET request to the **{orgId}/products/{productId}/configurations/{configId}** resource retrieves information for a defined product configuration. The body of the response contains the configuration information in JSON format.
-
-```json
-   GET [UM_Server]/{orgId}/products/{productId}/configurations/{configId}
+```
+GET [UM_Server]/{orgId}/products/{productId}/configurations/{configId}
```

Comment:
Could do this now: get rid of all occurrences of [UM_Server] and replace with actual url.

Reply:
@jabogart Can you take on the changes regarding this?